package/network/config/ltq-vdsl-app/files/10_atm.sh

```diff
@@ -14,3 +14,8 @@ fi
 modprobe ltq_atm_vr9
 
 /etc/init.d/br2684ctl reload
+
+sleep 2
```

Comment:
That looks error prone to race conditions. Can't we use something hotplug based?